grants/invariant/data.md

```diff
@@ -0,0 +1,37 @@
+---
+Grant-Number: "N/A",
+Name: "Invariant",
+Service-Provider: "",
+Status: "In progress",
+Grant-Link: "n/a",
+Website: "",
+Grant-Size: "Medium",
+Total-Milestones: "7",
+Current-Milestone: "M2",
```

Comment:
Could you check this please – current Milestone is mentioned as M2, however, in the individual Milestones list, M1 is marked as 'In progress'?